lib/libvmod_blob/vmod_blob.c

```diff
@@ -31,11 +31,21 @@
 
 #include "cache/cache.h"
 
-#include "vcc_if.h"
 #include "vmod_blob.h"
 
 #define VMOD_BLOB_TYPE 0xfade4faa
```

Comment:
Note: this comment is *not* about the PR, because the same is true in master.

Looking at the code I noticed that the magic numbers used in vmod_blob.c are 0xfade4faa, 0xfade4fa9 and 0xfade4fa0. They're too similar, and it defeats the purpose. If a struct is in fact corrupted at just the right byte, the CHECK_OBJ test might pass where it shouldn't.